hasura/graphql-engine
# console: persist page size in local storage

### Description

This PR makes page size persistent by storing it in local storage.

### Changelog


### Affected components


### Related Issues

https://github.com/hasura/graphql-engine/issues/3849

### Solution and Design

Added key `data:pageSize` to LS.

### Ideas

A good starting point to refactor local storage utils?

### Steps to test and verify

Go to `data` -> `table`. Change number of rows. Refresh page. The number of rows should be the same.
